src/components/pages/schedule/calendar/shiftDetails.jsx

```diff
@@ -97,13 +97,12 @@ export default function ShiftDetails({ vigil, setSelectVigil }) {
     const vigilRef = db.collection('vigils').doc(id);
     vigilRef.collection('shifts').add({
       address: vigil.address,
-      shiftStartTime, // TODO: This will need to be put into firebase as a timeStamp
-      shiftEndTime, // TODO: This will need to be put into firebase as a timeStamp
+      shiftStartTime,
```

Comment:
Can you put the TODOs (in just this file) back for this week because they aren't Done quite yet. (If you want to get rid of the code smells, just get rid of the TODO:, and leave the rest of the comment)

Reply:
Ok I will add them back in!